Lib/test/test_gc.py

```diff
@@ -766,6 +766,22 @@ def test_freeze(self):
         gc.unfreeze()
         self.assertEqual(gc.get_freeze_count(), 0)
 
+    def test_get_objects(self):
```

Comment:
This test would still work even if `generation` had no effect, right? I think the important test is to test generations that are being filtered out, so something more like:

```python
self.assertIn(l, gc.get_objects(generation=0))
gc.collect(generation=0)
self.assertNotIn(l, gc.get_objects(generation=0))
```

(Repeat as desired)

Reply:
Done in 604d331